cmd/batch-expire.go

```diff
@@ -668,6 +669,7 @@ func (r *BatchJobExpire) Start(ctx context.Context, api ObjectLayer, job BatchJo
 	ri.Complete = !failed && ri.ObjectsFailed == 0
 	ri.Failed = failed || ri.ObjectsFailed > 0
 	globalBatchJobsMetrics.save(job.ID, ri)
+	batchLogIf(ctx, ri.updateAfter(ctx, api, 0, job))
```

Comment:
We are already saving batch-expiry job stats once every 10s in here - https://github.com/minio/minio/pull/20098/files#diff-75c25e8d16842df078c17f4d7234993a6982cbae3ecbcb81a5d759f4f398d032R558.
We don't need this `ri.updateAfter` call here.

Reply:
Every *10* seconds? Please make it at least a minute.

Nothing will happen if it repeats a few after restarting. 10s just seems like a waste of IOPS.

Reply:
@krisis yes, I fixed a bug there, please take a look again

@klauspost I increased that to one minute